src/Algebra/Module/Construct/Zero.agda

```diff
@@ -38,11 +38,32 @@ module ℤero where
 open ℤero
```

Comment:
Is this directive now also redundant?

Reply:
I'm not sure it wasn't redundant to begin with, but either way I'll remove it

Reply:
(and I agree: it was redundant before! My mistake, then as now!)